docs/NOTIFICATIONS.md

```diff
@@ -0,0 +1,49 @@
+# NearIt-UI for notifications
+### Heads-up notifications for in-app content
+NearIt-UI can manage and show in-app content (read more about background and foreground notifications [here](http://nearit-android.readthedocs.io/en/latest/in-app-content/))
```

Comment:
"NearIt-UI can be set to manage and show notifications for foreground content (read more...)"

Reply:
done